.lintrunner.toml

```diff
@@ -1587,6 +1590,7 @@ exclude_patterns = [
     'test/test_view_ops.py',
     'test/test_vulkan.py',
     'test/test_xnnpack_integration.py',
+    'test/torch_np/numpy_test/**/*.py',
```

Comment:
One by one list these out, so new files get UFMT'ed

Reply:
I could do it, it's only that it doesn't add much value, since we didn't write those tests, and we just want to replicate whatever NumPy was doing, so adding plenty of `noqa`s did not seem to add too much value.

Reply:
This is UFMT, not FLAKE8; the formatting is automatic.

That being said, I suppose not gratuitously changing the formatting from the numpy tests is a good reason, I suppose.